src/main/scala/com/databricks/spark/xml/package.scala

```diff
@@ -19,10 +19,64 @@ import scala.collection.Map
 
 import org.apache.hadoop.io.compress.CompressionCodec
 
+import org.apache.spark.annotation.Experimental
 import org.apache.spark.sql._
-import com.databricks.spark.xml.util.XmlFile
+import org.apache.spark.sql.catalyst.expressions.Expression
+import org.apache.spark.sql.catalyst.parser.CatalystSqlParser
+import org.apache.spark.sql.types.StructType
+
+import com.databricks.spark.xml.util.{InferSchema, XmlFile}
 
 package object xml {
```

Comment:
Maybe we should create a package called `functions.scala` under this `xml` directory to match it to `avro`'s functions or other functions in Apache Spark.